typify-impl/tests/github.out

```diff
@@ -39802,6 +39817,11 @@ impl std::convert::TryFrom<String> for IssueCommentDeletedIssueActiveLockReason
 #[doc = "    },"]
 #[doc = "    {"]
 #[doc = "      \"$ref\": \"#/definitions/user\""]
+#[doc = "    },"]
+#[doc = "    {"]
+#[doc = "      \"not\": {"]
+#[doc = "        \"type\": \"null\""]
+#[doc = "      }"]
 #[doc = "    }"]
```

Comment:
this is wrong, but I'm not sure if it's much wronger. It would be good to come up with a simple test case for this

Reply:
https://github.com/oxidecomputer/typify/issues/487